.gitignore

```diff
@@ -23,6 +23,11 @@
 hs_err_pid*
 /bin/
 
-# Build-system generated directories
-/build/
-/dist/
+# Build Folders
+build/
+dist/
```

Comment:
Is there a reason these files no longer follow the convention of / beforehand? Just noticed that build and dist are now also changed

Reply:
there is not a reason for that. happened on accident when I duplicated the lines with out realizing it. I can swap them back. Any other thoughts to put in that commit?